src/Model/Wijkanders.hs

```diff
@@ -24,18 +25,21 @@ import           Data.Thyme                               ( Day
                                                           , ymdMonth
                                                           , ymdDay
                                                           )
+import           Data.Thyme                               ( _ymdDay )
```

Comment:
Please fix the issue reported by hlint:

```
Found:
  import Data.Thyme (Day, gregorian, ymdMonth, ymdDay)
  import Data.Thyme (_ymdDay)
  
Perhaps:
  import Data.Thyme (Day, gregorian, ymdMonth, ymdDay, _ymdDay)
```

Reply:
It's fixed in 832e7020b. If nothing has broken in new and unexpected ways.